src/main/java/uk/gov/nationalarchives/pdi/step/jena/model/JenaModelStep.java

```diff
@@ -306,7 +308,7 @@ private void closeAndThrow(final Model model, final String message) throws Kettl
     }
 
     private Object convertSqlValueToRdf(final Object sqlValue, @Nullable final RDFDatatype rdfDatatype) {
-        if (rdfDatatype == null || rdfDatatype.getURI().equals("http://www.w3.org/2001/XMLSchema#string")) {
+        if (rdfDatatype == null || rdfDatatype.getURI().equals(XSD_NAMESPACE_IRI + "#string")) {
```

Comment:
I think we should use Jena's built-in vocabularies and datatype equality support.
See adamretter/kettle-jena-plugins#1.